sdk/nodejs/types/input.ts

```diff
@@ -29198,7 +29198,7 @@ export namespace iot {
         targetArn: pulumi.Input<string>;
     }
 
-    export interface TopicRuleSqs {
+    export interface TopicRuleSq {
```

Comment:
Another issue.

Reply:
Fixed (pinned to `sqs` in https://github.com/pulumi/pulumi-aws/pull/2615/commits/28e1921c4b75c6e88db8281312d4b817ef3e7825.